statement_test.go

```diff
@@ -636,9 +652,91 @@ func TestIsCreateTableDDL(t *testing.T) {
 		},
 	} {
 		t.Run(tt.desc, func(t *testing.T) {
-			if got := isCreateTableDDL(tt.ddl, tt.table); got != tt.want {
+			if got := isCreateTableDDL(tt.ddl, tt.schema, tt.table); got != tt.want {
 				t.Errorf("isCreateTableDDL(%q, %q) = %v, but want %v", tt.ddl, tt.table, got, tt.want)
 			}
 		})
 	}
 }
+
+func TestExtractSchemaAndTable(t *testing.T) {
+	for _, tt := range []struct {
+		desc   string
+		input  string
+		schema string
+		table  string
+	}{
+		{
+			desc:   "raw table",
+			input:  "table",
+			schema: "",
+			table:  "table",
+		},
+		{
+			desc:   "quoted table",
+			input:  "`table`",
+			schema: "",
+			table:  "table",
+		},
+		{
+			desc:   "FQN",
+			input:  "schema.table",
+			schema: "schema",
+			table:  "table",
+		},
+		{
+			desc:   "FQN with spaces",
+			input:  "schema . table",
+			schema: "schema",
+			table:  "table",
+		},
+		{
+			desc:   "FQN, both schema and table are quoted",
+			input:  "`schema`.`table`",
+			schema: "schema",
+			table:  "table",
+		},
+		{
+			desc:   "FQN with spaces, both schema and table are quoted",
+			input:  "`schema` . `table`",
+			schema: "schema",
+			table:  "table",
+		},
+		{
+			desc:   "FQN, only schema is quoted",
+			input:  "`schema`.table",
+			schema: "schema",
+			table:  "table",
+		},
+		{
+			desc:   "FQN with spaces, only schema is quoted",
+			input:  "`schema` . table",
+			schema: "schema",
+			table:  "table",
+		},
+		{
+			desc:   "FQN, only table is quoted",
+			input:  "schema.`table`",
+			schema: "schema",
+			table:  "table",
+		},
+		{
+			desc:   "FQN with spaces, only table is quoted",
+			input:  "schema . `table`",
+			schema: "schema",
+			table:  "table",
+		},
+		{
+			desc:   "whole quoted FQN",
+			input:  "`schema.table`",
+			schema: "schema",
+			table:  "table",
+		},
+	} {
+		t.Run(tt.desc, func(t *testing.T) {
+			if schema, table := extractSchemaAndTable(tt.input); schema != tt.schema || table != tt.table {
+				t.Errorf("isCreateTableDDL(%q) = (%v, %v), but want (%v, %v)", tt.input, schema, table, tt.schema, tt.table)
```

Comment:
s/isCreateTableDDL/extractSchemaAndTable/

Reply:
Thanks!
fixed in https://github.com/cloudspannerecosystem/spanner-cli/pull/178/commits/c2cb1c336aa974ba57c451225f48bc5990cce68b